umm-spec-lib/src/cmr/umm_spec/spatial_conversion.clj

```diff
@@ -20,30 +21,37 @@
    "WRS-1"
    "WRS-2"])
 
+(def upcase-valid-tile-identification-system-names
+  "Upcase values to allow for case-insenitive validation"
+  (map util/safe-uppercase valid-tile-identification-system-names))
+
 (defn tile-id-system-name-is-valid?
   "Return whether or not the given TileIdentificationSystemName is one of the
    valid-tile-identification-system-names"
   [tile-system-id-name]
-  (some #(= tile-system-id-name %) valid-tile-identification-system-names))
+  (some #(= (util/safe-uppercase tile-system-id-name) %) upcase-valid-tile-identification-system-names))
 
 (defn translate-tile-id-system-name
   "Return nil or equivalent value if the given name does not match any
    in the list of valid ones"
   [tile-identification-system-name]
   (when (tile-id-system-name-is-valid? tile-identification-system-name)
```

Comment:
After you add the match-enum-case, there is no need to check if tile-id-system-name-is-valid? anymore. If the value doesn't have a case insensitive match in the enum list, it will return nil.

Reply:
This function is used in other places, so I'd rather not delete it

Reply:
I'm just asking you to drop the tile-id-system-name-is-valid check.  You already remove the function based on your response to my previous comment.